Artifacts/MsSql/Data/Security/2000-AdminApp-ClaimSets.sql

```diff
@@ -55,19 +60,42 @@ SELECT @actionId = ActionId FROM dbo.Actions WHERE ActionName = @actionName
 SELECT @claimSetId = ClaimSetId FROM dbo.ClaimSets WHERE ClaimSetName = @claimSetName
```

Comment:
SELECT @actionId = ActionId FROM dbo.Actions WHERE ActionName = @actionName  . Where is set value for @actionName variable